src/SUMMARY.md

```diff
@@ -21,21 +21,29 @@
     * [Write Document](contribute-to-tidb/write-document.md)
 
 * [Understand TiDB](understand-tidb/introduction.md)
-    * [MySQL protocol & session management](understand-tidb/mysql-protocol-and-session-management.md)
-    * [The lifecycle of a SQL](understand-tidb/the-lifecycle-of-a-sql.md)
-    * [Privilege management](understand-tidb/privilege-management.md)
-    * [DDL](understand-tidb/ddl.md)
-    * [DML](understand-tidb/dml.md)
-    * [Transaction processing](understand-tidb/transaction-processing.md)
-    * [Overview of planner](understand-tidb/overview-of-planner.md)
-    * [Logical optimization & rules](understand-tidb/logical-optimization-and-rules.md)
-    * [Physical optimization](understand-tidb/physical-optimization.md)
-    * [Table statistics](understand-tidb/table-statistics.md)
-    * [Plan cache](understand-tidb/plan-cache.md)
-    * [SQL hints and plan management](understand-tidb/sql-plan-management.md)
-    * [Overview of the execution engine](understand-tidb/overview-of-the-execution-engine.md)
-    * [Retrieving data from the storage layer](understand-tidb/communicate-with-the-storage.md)
-    * [MVCC garbage collection](understand-tidb/mvcc-garbage-collection.md)
+    * [The Lifecycle of a Statement](understand-tidb/the-lifecycle-of-a-statement.md)
+        * [DDL](understand-tidb/ddl.md)
+        * [DML](understand-tidb/dml.md)
+        * [DQL](understand-tidb/dql.md)
+    * [Parser](understand-tidb/parser.md)
+    * [Planner](understand-tidb/planner.md)
+        * [Table Statistics](understand-tidb/table-statistics.md)
+        * [Rule-based Optimization](understand-tidb/rbo.md)
+        * [Cost-based Optimization](understand-tidb/cbo.md)
+        * [Plan Cache](understand-tidb/plan-cache.md)
+        * [SQL Plan Management](understand-tidb/sql-plan-management.md)
+    * [Execution](understand-tidb/execution.md)
+        * [Hash Aggregation](understand-tidb/hash-aggregation.md)
```

Comment:
```suggestion
        * [Execution Framework](understand-tidb/execution-framework.md)
        * [Implementation of Vectorized Execution](understand-tidb/implementation-of-vectorized-execution.md)
        * [Memory Management Mechanism](understadn-tidb/memory-management-mechanism.md)
        * [Implementation of Typical Operators](understand-tidb/implementation-of-typical-operators.md)
```

Reply:
Shall we cover how to communicate with storage engine in Execution chapter?

Reply:
.. or it will be covered by `Execution Framework` or `Implementation of Typical Operators` (`TableReader`)?

Reply:
I think we may make it as a subpart of `Implementation of Typical Operators`.

Reply:
Make sense, integrating...

Reply:
@XuHuaiyu updated. You can approve the PR if there is no further comments.